src/features/views/l10n/messageIds.ts

```diff
@@ -55,6 +55,9 @@ export default makeMessages('feat.views', {
         '{count, plural, =1 {1 solved issue} other {# solved issues}}'
       ),
     },
+    personTag: {
+      emptyValue: m('Empty value'),
```

Comment:
I thought about it, went with this, but yes agree it might not be The One. Thought about both "Empty tag" and "No value". Other potential candidates in my head: "No value added", "No value yet".